src/core/action-creators.ts

```diff
@@ -65,33 +80,33 @@ export const sync = (state, log) => ({
  * @param {object} state - The state to restore.
  * @param {Array} deltalog - A log delta.
  */
-export const update = (state, deltalog) => ({
-  type: Actions.UPDATE,
+export const update = (state: any, deltalog: any[]) => ({
```

Comment:
Let's use the `State` type instead of `any` here?  There are a few other spots in this file.

Reply:
Done :+1: